templates/components/itilobject/timeline/form_task.html.twig

```diff
@@ -154,7 +155,7 @@
    {% else %}
       <div class="itiltask">
          <form name="asset_form" style="width: 100%;" class="d-flex flex-column" method="post"
-               action="{{ subitem.getFormURL() }}" enctype="multipart/form-data" data-track-changes="true" data-submit-once>
+               action="{{ subitem.getFormURL() }}"  enctype="multipart/form-data" data-track-changes="true" data-submit-once {{ formoptions|raw }}>
```

Comment:
We should try to remove these `{{ formoptions|raw }}`. It does not seems secure to me to add attributes values without any escaping logic on `form` elements.

```suggestion
               action="{{ subitem.getFormURL() }}"  enctype="multipart/form-data" data-track-changes="true" data-submit-once {% if form_id is defined %}id="{{ form_id }}"{% endif %}>
```

Reply:
I agree we should not use raw as is.

Reply:
This parameter should be consistent with its others implementation.
It can be changed in another PR but it must be done everywhere, not just for for this use case.

Reply:
OK if another PR fixes this globally.